internal/cache/file/cache_handle_test.go

```diff
@@ -83,7 +83,8 @@ func (cht *cacheHandleTest) addTestFileInfoEntryInCache() {
 func (cht *cacheHandleTest) verifyContentRead(readStartOffset int64, expectedContent []byte) {
 	fileStat, err := os.Stat(cht.fileSpec.Path)
 	AssertEq(nil, err)
-	AssertEq(cht.fileSpec.Perm, fileStat.Mode())
+	AssertEq(cht.fileSpec.FilePerm, fileStat.Mode())
+	AssertEq(cht.fileSpec.DirPerm, 0700)
```

Comment:
Since we set cht.fileSpec.DirPerm, so it will not help if we assert that with a fixed value. 
What we need to do is get the directory of fileSpec.Path using path/filepath.Dir(fileSpec.Path), then do os.Stat(dirPath) and then check if it's same as cht.fileSpec.DirPerm.

Reply:
done